.github/ISSUE_TEMPLATE/ticket_template1.md

```diff
@@ -0,0 +1,27 @@
+🚨Please review the [guidelines for ticket creation](../HOWTOWRITEATICKETS.md) if needed
+
+## Problem / Motivation
+Describe your issue here. You can use the user story format.
+- As a ... Who?
+- I would like ... what?
+- So that ... why?
```

Comment:
My problem with the user-story format is that people are generally _really_ bad at it ('As a user, I want to log in so that I can access the application').

I would much prefer asking questions here.

Reply:
Here the user-story format is just a suggestion. What matters is that the Who? What? and Why?  are captured. I will make that clearer.